pkg/sql/create_role.go

```diff
@@ -299,3 +296,26 @@ func (ua *userNameInfo) resolveUsername() (string, error) {
 
 	return normalizedUsername, nil
 }
+
+func (p *planner) checkPasswordAndGetHash(
+	ctx context.Context, password string,
+) (hashedPassword []byte, err error) {
+	if password == "" {
+		return hashedPassword, security.ErrEmptyPassword
+	}
+	hashedPassword, err = security.HashPassword(password)
+	if err != nil {
+		return nil, err
+	}
+
+	if hashedPassword == nil {
+		// v20.1 and below crash during authentication if they find a NULL value
+		// in system.users.hashedPassword. v20.2 and above handle this correctly,
+		// but we need to maintain mixed version compatibility for at least one
+		// release.
+		// TODO(nvanbenschoten): remove this for v21.1.
+		hashedPassword = []byte{}
+	}
+
+	return
```

Comment:
Wait shouldn't this return hashedPassword or a return inside the if statement?

Reply:
I don't understand your question. Can you rephrase?

Reply:
Ah nvm I see that the return values are named, ignore my comment. 

Not a huge fan of named return values with an empty return statement though.